plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -211,9 +213,8 @@ func (p *antreaOctantPlugin) actionHandler(request *service.ActionRequest) error
 		ctx := context.Background()
 		tfOld, _ := p.client.OpsV1alpha1().Traceflows().Get(ctx, tfName, v1.GetOptions{})
 		if tfOld.Name == tfName {
-			log.Printf("Duplicate traceflow \"%s\": same source pod and destination pod in less than one second"+
-				": %+v. ", tfName, tfOld)
-			return nil
+			log.Printf("Invalid user input, CRD creation or Traceflow request may fail: "+
```

Comment:
When iterating this feature, I suggest adding a suffix of a random string to save the step, or maybe we could just set "generateName" instead of "name" to ask apiserver to create a unique name.

Reply:
Should I fix it in this patch or fix it later?